drakcore/package/build.sh

```diff
@@ -3,13 +3,17 @@
 # Directory with .deb files
 OUTDIR=$(pwd)/out
 # Base image for building
-BASEIMAGE=ubuntu:18.04
+if [ "$BASEIMAGE" = "" ]
+then
+    BASEIMAGE=debian:buster
+fi
+
 # Current user - to make current user owner of output .deb files.
 FILE_OWNER=$(id -u):$(id -g)
 
 echo "Building $BASEIMAGE based image into $OUTDIR"
 docker build -f drakcore/package/Dockerfile \
-	     --build-arg IMAGE=$BASEIMAGE \
+             --build-arg IMAGE=$BASEIMAGE \
              -t deb-build-web . && \
 docker run --rm \
```

Comment:
why random indentation eveywhere tho

Reply:
tabs vs spaces, this was the only space in the file